### ISSUE TITLE
Potential rewards sometimes are not displayed according to slider value. 

### ISSUE THREAD

**Issue by Klobbinger** [input-output-hk/daedalus#2688]
See screenshot. The amount on the slider was manually chosen. This happens sporadically. Sometimes it will show the correct rewards estimation, other times it'll show the amount corresponding to the default 1000 ADA.

![grafik](https://user-images.githubusercontent.com/67224145/135761236-b94ddf90-40e4-43ac-ae42-bc41c0a536ea.png)

**Comment by miorsufianiohk** [comment]
Dear @Klobbinger ,

Thank you for letting us know. We will investigate and let you know in due course.

Kind regards,
Mior

**Comment by miorsufianiohk** [comment]
Hi @Klobbinger ,

I hope that you are well. Do you mind giving it another go please and see if you could reproduce this issue on the latest Daedalus (please note that the pool estimated rewards will be treated in its own ticket). .

Many thanks,
Mior

**Comment by PhilippeLeLong** [comment]
I was able to replicate the bug once after initial start up. After that, I couldn't replicate it. I will try again tomorrow and report back. 
![grafik](https://user-images.githubusercontent.com/67224145/146596872-a51e6ec2-76dc-4482-9276-1a04b4a2dcb7.png)

**Comment by PhilippeLeLong** [comment]
So it seems like this only happens after starting Daedalus and going to the stake pool tab. first time the slider is adjusted it does not update the expected rewards. After moving the slider a second time the correct value is shown. 

![grafik](https://user-images.githubusercontent.com/67224145/146639472-a24bccea-60c0-41e1-b685-7f04c8e18e6a.png)
![grafik](https://user-images.githubusercontent.com/67224145/146639480-26eaa6a0-17f9-4e18-af65-c4222e97aae4.png)

**Comment by miorsufianiohk** [comment]
Hi @PhilippeLeLong ,

Thank you for your replies. We have been trying to reproduce the issue without any success. We tested on multiple platforms including Windows, Mac and Linux on at least two networks which are Mainnet and Testnet. I have been wondering if you could still reproduce it? If possible, could you kindly let us know the following details please:

1)Your operating system and version
2)RAM size
4)The hard disk space you currently have left in the machine
4)The Daedalus that produces the issue, ie (Mainnet/Flight or Testnet) and the build number
5)Is the Daedalus a fresh install or an upgrade?
6)Any other details that you feel might be beneficial for us

In the meantime, we will continue with our investigation and look forward to receiving the above information from you.

Kind regards,
Mior

**Comment by PhilippeLeLong** [comment]
1) Windows 10 Home, 21H1, Build 19043.1415
2) 16GB
3) 40 GB and 500 GB. Daedalus folder is on D: drive. Same issue though before when everything was on C:
4) Mainnet Build 20052
5) Upgrade from previous versions, fresh install in October 2020, don't know which version. 
6) Steps to reproduce:

1. Open Daedalus wait for connection and full sync
2. Go to Stake Pools tab
3. click list view
4. move slider once and wait for calculation to be finished. Result see attached screnshot.
5. Any other action afterwards that updates the stake pool list again will result in the correct rewards shown.

Hope that helps.

![grafik](https://user-images.githubusercontent.com/67224145/148220913-1d048b55-daf5-4f8b-be3b-68382fbca572.png)

**Comment by PhilippeLeLong** [comment]
Installed Daedalus on a new PC. Latest version 4.9.1 and the problem still persists.

**Comment by PhilippeLeLong** [comment]
![image](https://user-images.githubusercontent.com/67224145/173194755-aa522ec3-7a9e-42f4-b328-1d26ef981845.png)
Daedalus version 4.11.0

Same faulty behavior. The first time after opening Daedalus and going to the stake pools tab and sliding to a custom amount, the potential rewards are calculated incorrectly. Only after adjusting the slider again the UI will show correct values.

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
